app/controllers/home_controller.rb

```diff
@@ -19,15 +19,17 @@ def dashboard
       user_id: current_user.id,
       recent: true,
       include_private: true,
-      include_adult: !current_user.sfw_filter?
+      include_adult: !current_user.sfw_filter?,
```

Comment:
It's a common practice to put a comma on the last hash item because if you have to add another value later on, you'll have two lines on the git diff, the line added and the comma on the previous line